core/src/main/java/io/undertow/conduits/InflatingStreamSourceConduit.java

```diff
@@ -72,18 +74,52 @@ public InflatingStreamSourceConduit(
             ObjectPool<Inflater> inflaterPool) {
         super(next);
```

Comment:
How about using this( ... ) constructor as above?

Reply:
We could, but I think JDK would not like that: "The constructor InflatingStreamSourceConduit(StreamSourceConduit) is undefined"